storage/object/api-cli/object-operations.mdx

```diff
@@ -323,6 +323,15 @@ Connection: close
 ``` 
 aws s3api put-object --bucket BucketName --key dir-1/ObjectName --body ObjectName
 ``` 
+
+<Message type="note">
+
+  To define the [storage class](/storage/object/concepts/#storage-class) of the object directly upon creation, add the `x-amz-storage-class` header and specify one of the following classes: `STANDARD`, `ONEZONE_IA`, `GLACIER`. Example: `x-amz-storage-class: ONEZONE_IA`. 
+  
```

Comment:
Since we're showing examples of both the `awscli s3api` calls and the raw HTTP requests on this page, shouldn't we provide an example for both in regards to the storage class?

```suggestion
  To define the [storage class](/storage/object/concepts/#storage-class) of the object directly upon creation, use the `--storage-class <class>` option with `awscli` or add the `x-amz-storage-class <class>` header if you're using the S3 API directly. You can specify one of the following classes: `STANDARD`, `ONEZONE_IA`, `GLACIER`. Example: `x-amz-storage-class: ONEZONE_IA`. 
  
```

(https://docs.aws.amazon.com/cli/latest/reference/s3api/put-object.html)